inference-engine/src/inference_engine/ie_core.cpp

```diff
@@ -625,6 +590,12 @@ class Core::Impl : public ICore {
         return LoadNetworkImpl(network, deviceName, config, nullptr);
     }
 
+    // TODO: mnosov: do we really need this in ICore class (pure virtual method)?
+    ExecutableNetwork LoadNetwork(const std::string& modelPath, const std::string& deviceName,
```

Comment:
> do we really need this in ICore class (pure virtual method)?

Usually, method is added to `ICore` interface, if any plugin need to call this method. Otherwise, it can be a method of `Core::Impl` only.

Reply:
So, we'll not need adding this to ICore, right? I don't see cases where this method can be called by plugins

Reply:
Hm, only if it's re-used in `MULTI` plugin (likely we need to extend Plugin API interface in this case). In this case `MULTI` will pass `fileName` directly to this new method `ICore::LoadNetwork(fileName, ...)` and loading will be performed faster.. 

But I think it can done as a separate task in a separate PR..